src/util/VehicleSpawner.cpp

```diff
@@ -2,15 +2,14 @@
 
 namespace YimMenu
 {
-	extern bool SpawnVehicle(std::string model_name, int player_ped_id)
+	extern int SpawnVehicle(std::string model_name, int player_ped_id, std::optional<Vector3> coords)
```

Comment:
The player_ped_id looks out of place here. You can gather such information before invoking this function. POC 
`auto location = ENTITY::GET_OFFSET_FROM_ENTITY_IN_WORLD_COORDS(ANYPED, 0, -3, 0)
SpawnVehicle(anyveh, location) `

Reply:
I'd advise you to adhere to SOLID principles and single responsibility rules. This function could prove to be essential so its function should be relevant to its nature. Any expansions can be done in a manner of wrapper functions or simple functional programming.

Reply:
> The player_ped_id looks out of place here. You can gather such information before invoking this function. POC 
> 
> `auto location = ENTITY::GET_OFFSET_FROM_ENTITY_IN_WORLD_COORDS(ANYPED, 0, -3, 0)
> 
> SpawnVehicle(anyveh, location) `

Okay, I will change this and push my changes in about an hour.